src/README.md

```diff
@@ -10,6 +10,7 @@ The structure and contents of the examples directory are as follows:
 
  * [nist.gov/SP800-53/rev4](nist.gov/SP800-53/rev4): This directory contains OSCAL examples of the catalog, and low, moderate, and high baselines defined by NIST Special Publication (SP) 800-53 Revision 4.
  * [nist.gov/SP800-53/rev5](nist.gov/SP800-53/rev5): This directory contains OSCAL examples of the catalog, and low, moderate, and high baselines defined by NIST Special Publication (SP) 800-53B Revision 5 and SP 800-53B Revision 5 respectively.
+   * [nist.gov/SP800-53/rev5/draft](nist.gov/SP800-53/rev5/draft): Earlier releases of draft content extractions of the Final Public Draft (FPD) version of the Revision 5 catalog and profiles are available - now superseded, these are provided for reference only.

```

Comment:
Suggest this statement to be updated to a content that reads a little better and that is similar to the other descriptions . E.g. 
This directory contains the OSCAL Final Public Draft (FPD) version of SP 800-53 Revision 5 catalog and SP 800-53B profiles. This content is now superseded, and the files are provided for reference only.